packages/combobox/src/types.ts

```diff
@@ -8,7 +8,7 @@
 import { IUseFieldReturnValue } from '@zendeskgarden/container-field';
 import { HTMLProps, ReactNode, RefObject } from 'react';
 
-export type OptionValue = string | object;
+export type OptionValue = string;
```

Comment:
I liked the idea of retaining this type definition for internal use in spotting where `value`s are referenced in code.

Reply:
The problem is that this doesn't play nice with the generated docs. I would favor removing entirely so that the updated docs are very clear that the value is now `string` only.